README.md

```diff
@@ -10,12 +10,12 @@ You can add the extension on your project as follows, replacing, at least, `base
 
 ```python
 import talisker.requests
-from canonicalwebteam.discourse_docs import DiscourseDocs, DiscourseAPI
+from canonicalwebteam.discourse_docs import Discourse, DiscourseAPI
```

Comment:
I think we want to rename the module `canonicalwebteam.discourse` - but if you're worried about how to do that then we can merge this PR first and we can follow up with a rename PR.